tests/datasets/test_geo.py

```diff
@@ -162,6 +163,56 @@ def test_abstract(self) -> None:
             VisionDataset()  # type: ignore[abstract]
 
 
+class TestVisionClassificationDataset:
+    @pytest.fixture(scope="class")
+    def dataset(self) -> VisionClassificationDataset:
+        root = os.path.join("tests", "data", "visionclassificationdataset")
```

Comment:
You can use the below fixture in this fixture so you don't have to repeat yourself